pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions_test.go

```diff
@@ -369,7 +369,8 @@ func newPlugin(t *testing.T) framework.Plugin {
 }
 
 func newPluginWithListers(t *testing.T, pods []*v1.Pod, nodes []*v1.Node, pvcs []*v1.PersistentVolumeClaim, enableReadWriteOncePod bool) framework.Plugin {
-	ctx := context.Background()
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
```

Comment:
I think maybe it would be better to init `ctx` outside and add it to the function as an input variable. It allows us to cancel the `ctx` outside. WDYT? This is maybe not relevant to this pr. And I can submit other PR to deal with this.

Reply:
Good point. It's also why we cannot run `defer cancel()` here.

> This is maybe not relevant to this pr. And I can submit other PR to deal with this.

I think it's related and will include it in this PR.

Reply:
sg thanks